int8 numSongs
---
string playlist